django/contrib/admindocs/views.py

```diff
@@ -126,13 +127,23 @@ def get_context_data(self, **kwargs):
 class ViewIndexView(BaseAdminDocsView):
     template_name = 'admin_doc/view_index.html'
 
+    @staticmethod
+    def _get_full_name(func):
+        mod_name = func.__module__
+        if six.PY3:
+            return '%s.%s' % (mod_name, func.__qualname__)
+        else:
+            # PY2 does not support __qualname__ (#27018)
```

Comment:
We don't typically include ticket numbers except for obscure issues where the context of the ticket offer some additional information that can't be explained in comments. I think you can remove it for this issue (both in code and tests).